bootstrap_sdk

```diff
@@ -233,9 +233,11 @@ if [[ "$STAGES" =~ stage4 ]]; then
     sign_and_upload_files "packages" "${def_upload_path}" "pkgs/" \
         "${BINPKGS}"/*
 
-    # Upload the SDK toolchain packages
-    sign_and_upload_files "cross toolchain packages" "${def_upload_path}" \
-        "toolchain/" "${BINPKGS}/crossdev"/*
+    if [ -d "${BINPKGS}/crossdev" ]; then
+        # Upload the SDK toolchain packages
+        sign_and_upload_files "cross toolchain packages" "${def_upload_path}" \
+            "toolchain/" "${BINPKGS}/crossdev"/
```

Comment:
Should `toolchain/` still be uploaded?

Reply:
Also I believe the `/*` suffix was there because the subfolders should be uploaded, or?

Reply:
> Should `toolchain/` still be uploaded?

Got it, I see now that this is the source argument, ok. Still I think the `/*` should stay - there was no test run with `all-full`.

Reply:
Pushed a fix.
CI running http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/3244/cldsv

Reply:
Even without the `/*` subdirs get uploaded: sign_and_upload_files does this:
```
for file in "$@"; do
  ...
  for sigfile in $(find "${file}" ! -type d); do```

```

Reply:
I wonder whether this results in a different target directory structure.